modules/ROOT/partials/release-notes/6.0-release-notes-summary.adoc

```diff
@@ -242,7 +242,7 @@ The following elements were previously deprecated and have, with this release, b
 
 | `non_empty_elements`                  | Schema option |
 
-| `padd_empty_with_br`                  | Option        |
+| `padd_empty_with_br`                  | Option        | Replaced, as of {productname} 6.6.1, by xref:content-filtering.adoc#pad_empty_with_br[`pad_empty_with_br`].
```

Comment:
@brian-forte should this be `Replaced, as of {productname} 6.6.2`? rather than `6.6.1`.

```suggestion
| `padd_empty_with_br`                  | Option        | Replaced, as of {productname} 6.6.2, by xref:content-filtering.adoc#pad_empty_with_br[`pad_empty_with_br`].
```